Add Blink Integration (#1597)
DataDog/integrations-extras: blink/README.md

# BlinkOps Integration

## Overview

> The Datadog integration for [Blink][1] enables you to trigger automations in response to new incidents. You can also create, update, and view incidents and events in the DataDog event stream.

Reviewer: ```suggestion
The Datadog integration for [Blink][1] enables you to trigger automations in response to new incidents. You can also create, update, and view incidents and events in the Datadog event stream.
```

## Setup

Visit [our documentation][2] for details on how to integrate BlinkOps with Datadog.

## Data Collected

### Events

> BlinkOps lets you send events, incidents, and monitors your Datadog event stream from a Blink automation.

Reviewer: ```suggestion
BlinkOps lets you send events and incidents, and monitors your Datadog event stream from a Blink automation.
```

Searching and updating incidents is also supported with the DataDog integration in Blink.

### Metrics

> BlinkOps does not include any metrics, however you can query and list metrics from your DataDog environment for use in Blink automations.

Reviewer 2: ```suggestion
BlinkOps does not include any metrics, however you can query and list metrics from your Datadog environment for use in Blink automations.
```

# BlinkOps Integration

## Overview

> The [Blink][1] integration enables you to:

Reviewer 3: Hey @liavt - thanks for making the changes! In addition to what you currently have, It would be helpful if you could include 1-2 sentences about the integration and the value.  We typically like to have the following:

1. 1-2 sentences about the integration so customers know what it is
2. What customers will get from this integration (what you have now covers this) 
3. Data collected (What you have now covers this)

Author: Hi, I added some additional information about our integration

> ## Overview
> 
> The Datadog integration for [Blink][1] enables you to trigger automations in response to new incidents. You can also create, update, and view incidents and events in the Datadog event stream.

Reviewer 3: ```suggestion
## Overview

The [Blink][1] integration enables you to:
1. Trigger automations within Blink in response to new Datadog incidents 
2. Create and update Datadog incidents in Blink 
3. View any created incidents or events in the Datadog event stream
```

Reviewer 3: Included some information on the integration but feel free to edit if I misunderstood.

Author: Information looks good, committed the change

# BlinkOps Integration

## Overview

> The [Blink][1] integration is a collection of no-code actions and triggers usable in Blink Automations. Instead of writing manual API calls and scripts, you can use Blink's visual drag-and-drop editor to quickly automate your Datadog workspace, or use one of the hundreds of existing pre-made automations from the [library][2].

Reviewer 3: Can you clarify on what you mean by drag and drop editor? We want to make sure the description is as clear as possible for customers.

Author: We allow users to drag and drop "actions" into a automation which represents a workflow, so our integration with Datadog is that we have Datadog actions that you can insert. I added some clarification to the readme about this but let me know if it is still unclear

> The [Blink][1] integration is a collection of no-code actions and triggers usable in Blink Automations. Instead of writing manual API calls and scripts, you can use Blink's visual editor to drag-and-drop Datadog API actions into your workflow. You can use also one of the hundreds of existing pre-made automations from the [library][2].

Reviewer 3: ```suggestion
The [Blink][1] integration is a collection of no-code actions and triggers usable in Blink Automations. Instead of writing manual API calls and scripts, you can use Blink's visual editor to drag-and-drop Datadog API actions into your workflow or use one of the hundreds of existing pre-made automations from the [library][2].
```

This integration enables you to:
1. Trigger automations within Blink in response to new Datadog incidents 
2. Create and update Datadog incidents in Blink 

> 3. View any created incidents or events in the Datadog event stream

Reviewer 3: Do you mean any created incidents or events in Blink are viewable in Datadog?

Author: You can view created incidents or events in Blink, even if they weren't created by Blink themselves, since we use the API to get the event stream. I updated the readme to clarify this point
